src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -318,35 +319,6 @@ class ElytraFlight : Module() {
         }
     }
 
-    /**
-     * @return true if there is liquid below
-     */
-    private fun checkForLiquid(): Boolean {
-        return getGroundPosY(true) == -1.0f
-    }
-
-    /**
-     * Get the height of the ground surface below, and check for liquid if [checkLiquid] is true
-     *
-     * @return The y position of the ground surface, -1.0f if found liquid below and [checkLiquid] is true
-     */
-    private fun getGroundPosY(checkLiquid: Boolean): Float {
```

Comment:
oh uhhhh the vec helper already had this function, oops

Reply:
This one uses player bounding box to check the ground position